scripts/us_census/pep/monthly_population/download.py

```diff
@@ -0,0 +1,153 @@
+""" A Script to download USA Census PEP monthly population data

```

Comment:
Please add a licence header with 2022, similar to
https://github.com/datacommonsorg/data/blob/master/scripts/us_census/acs5yr/subject_tables/common/data_loader.py

Reply:
Sure, Ajai.